todo/models.py

```diff
@@ -1,3 +1,61 @@
 from django.db import models
-
+from django.utils.translation import gettext_lazy as _  # renaming variables
 # Create your models here.
+
+class Category(models.Model):
+    name = models.CharField(_('Category Name'), max_length=200,
+                            blank=False,
+                            null=True,
+                            help_text='Add categories to your task.')
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = 'Category'
+        verbose_name_plural = 'Categories'
+
+class Status(models.Model):
+    name = models.CharField(_('Status Name'), max_length=200,
+                            blank=False,
+                            null=True,
+                            help_text='Add status of your task.')
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = 'Status'
+        verbose_name_plural = 'Status'
+
+class Task(models.Model):
+    name = models.CharField(_('What\'s to be done'), max_length=250,
+                            blank=False,
+                            null=True,
+                            help_text='Add your todo task.')
+    date = models.DateField(_('Pick a date'),
+                            null=True,
+                            blank=False,
+                            help_text='Click the box and select date.')
+    time = models.DateField(_('Pick a time'),
```

Comment:
This should be a **TimeField** and not **DateField** @codefilterPH

Reply:
The model should possibly have an owner field pointing to user, so that we can filter base on user

Reply:
@DamaMichaelYohanna that correct. But we haven't merge your custom user model which will serve as a foreign-key. to the todo app

Reply:
@DamaMichaelYohanna  I'm waiting for your latest pull request before I will merge it.

Reply:
@Teams-Motion made a commit right now.

Reply:
Hello, you can check now I added the users to it

Reply:
I will add organization nest so a person can join one